app/Http/Controllers/Api/SignupsController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Rogue\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+use Rogue\Services\SignupService;
+use Rogue\Repositories\PhotoRepository;
+use Rogue\Http\Transformers\SignupTransformer;
+
+// use Rogue\Http\Requests;
+
+class SignupsController extends ApiController
+{
+    /**
+     * @var \League\Fractal\TransformerAbstract;
+     */
+    protected $transformer;
+
+    /**
+     * The signup service instance.
+     *
+     * @var Rogue\Services\SignupService
+     */
+    protected $signups;
+
+    /**
+     * The photo repository instance.
+     *
+     * @var Rogue\Repositories\PhotoRepository
+     */
+    protected $photo;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract  $posts
+     * @return void
+     */
+    public function __construct(SignupService $signups, PhotoRepository $photo)
+    {
+        $this->signups = $signups;
+        $this->photo = $photo;
+        $this->transformer = new SignupTransformer;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $transactionId = incrementTransactionId($request);
+
+        $signup = $this->signups->create($request->all(), $transactionId);
+
+        if ($signup) {
+            $code = '200';
```

Comment:
That's backwards! We should be using [`201 Created`](https://httpstatuses.com/201) when a new resource is created, and [`200 OK`](https://httpstatuses.com/200) for general "success" responses.